src/escalus_fresh.erl

```diff
@@ -16,7 +16,7 @@
 -define(MIN_UNREGISTER_TEMPO, 20000).
 
 
--type user_res() :: {atom(), integer()}.
```

Comment:
I think we could just use the original types instead of these aliases to improve readability.

Reply:
Indeed, good point.